packages/core/src/reporters/fix/components/Diff.tsx

```diff
@@ -27,7 +26,7 @@ export const Diff = ({ diff }: { diff: Difference }) => {
         ))}
       </Text>
     ) : (
-      <Text>- "files" set to []</Text>
+      <Text>- "files" updated</Text>
```

Comment:
If there are no `missingFiles` then how can this have updated? Or can the sorting order change as part of `crackle fix`? If so, maybe it'd be better to explicitly say that, or honestly say nothing as the order doesn't matter.

Reply:
The order might have changed or `"files"` didn't exist before. Better to let the user know when something changed, than not.